worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+    range_start = 0
+    range_end = 21
+    default = 0
+
+class Traps(Toggle):
+    """Swaps out the Big-O-Pants with Traps!"""
+    display_name = "Traps"
+
+class EnableMultiWorldDinoRoar(Toggle):
+    """Baby T-Rex Roar is lost across the MultiWorld. Other players need to help him learn to ROAR!"""
+    display_name = "Baby T-Rex Roar"
+
+class EnableNestsanity(Toggle):
+    """Eggs and feather nests give checks when you collect them for the first time. They behave as regular egg nests the other times."""
+    display_name = "Nestsanity"
+    
+class TrapsToNestRatio(Range):
+    """Select a percentage of all feather and egg nests items to be replaced with trap items
+    Requires Traps and Nestsanity to have an effect."""
+    display_name = "Traps to Nests Ratio"
+    range_start = 0
+    range_end = 100
+    default = 0
+    
+class GoldenEggsWeight(Range):
+    """The weight of Golden Eggs in the trap pool.
+    Requires Traps and Nestsanity to have an effect"""
+    display_name = "Golden Eggs Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+
+class TripTrapWeight(Range):
+    """The weight of Trip Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Trip Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class SlipTrapWeight(Range):
+    """The weight of Slip Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Slip Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class TransformTrapWeight(Range):
+    """The weight of Transform Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Transform Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class SquishTrapWeight(Range):
+    """The weight of Squish Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Squish Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 20
+    
+class KingJingalingHasJiggy(DefaultOnToggle):
+    """King Jingaling will always have a Jiggy for you."""
+    display_name = "King Jingaling Jiggy"
+
+class SkipPuzzles(DefaultOnToggle):
+    """Open world entrances without having to go to Jiggywiggy."""
+    display_name = "Skip Puzzles"
+
+class Backdoors(Toggle):
+    """Opens many one way switches on game start, allowing for more backdoor access to levels.
+    The following gates are preopened: MT>TDL, MT>HFP, GGM>WW, WW>TDL
+    For MT>TDL only the gate accessed from TDL's side is opened and for GGM>WW the boulders are still unexploded
+    The bridge from HFPs entrance is pre-moved to allow secondary access to clifftop
+    George is also pre-dropped to make HFP>JRL more accessible"""
+    display_name = "Open Backdoors"
+
+class OpenHag1(DefaultOnToggle):
+    """HAG 1 boss fight is opened when Cauldron Keep is opened. Only 55 jiggies are needed to win."""
```

Comment:
```suggestion
    """HAG 1 boss fight is opened when Cauldron Keep is opened. Only 55 Jiggies are needed to win."""
```
Consistent capitalization for Jiggy